docs/data/joy/components/alert/alert.md

```diff
@@ -64,12 +102,11 @@ Use the `startDecorator` and/or `endDecorator` props to insert actionable button
 
 ## Accessibility
```

Comment:
I think that the content should be synced with https://deploy-preview-34838--material-ui.netlify.app/material-ui/react-alert/#accessibility.